sample_factory/envs/atari/atari_utils.py

```diff
@@ -42,6 +42,8 @@ def atari_env_by_name(name):
 def make_atari_env(env_name, cfg, **kwargs):
     atari_spec = atari_env_by_name(env_name)
 
+    # to render atari, need to add render_mode
+    # env = gym.make(atari_spec.env_id, render_mode='human')
```

Comment:
This needs to be some kind of permanent fix later on, without manual commenting/uncommenting of code.
For example this can be achieved by enjoy script registering a different make_env function where this is set automatically

Reply:
Got it.